docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -6,94 +6,208 @@ description: "Efficiently allocate work through user tasks."
 
-1. Go back to your Camunda Platform 8 diagram and select the honeycomb icon and then **View user tasks** to take a look at your user tasks inside Tasklist.
-2. Select the open user task on the left panel of **Tasks**. In our example below, this is **Decide what's for dinner**.
-3. Next to **Assignee**, click **Claim** to claim the task.
-4. Once finished entering the appropriate information, click **Complete Task**.
-   ![complete a human task in Tasklist](./img/user-task-tasklist.png)
-5. On the left panel of **Tasks**, filter by **Completed** tasks to see your task has been finished.
+2. In Operate, you will see a visualization of the running process instance. Notice that a green **token** is waiting at the user task. This means that a task is waiting to be worked on in Tasklist.
+   <img src={OperateHumanTasks} alt="Process instance monitoring in Operate" />
 
-You can now navigate back to Operate and notice the process instance has continued, and the token has moved forward.
+:::tip
+In production, Operate is used to monitor both long-running and straight-through, high-throughput processes. In development environments, use Operate to confirm if the process flow works as expected. For faster in-place validation during development, use the [Play mode](/components/modeler/web-modeler/play-your-process.md).
+:::
 
-The token moves through the exclusive gateway (also called the XOR gateway), and is used to model the decision in the process. When the execution arrives at this gateway, all outgoing sequence flows are evaluated in the order in which they have been defined. The sequence flow which condition evaluates to ‘true’ (or which doesn’t have a condition set, conceptually having a ‘true’ value defined on the sequence flow) is selected for continuing the process.
+### Step 5: Complete a user task
 
-:::note
-Here, after implementing your gateway, is when [expressions](/components/concepts/expressions.md) will become useful in accessing [variables](/components/concepts/variables.md) and calculating their value to move forward in a process.
+When the process instance arrives at the user task, a new job is created. The process instance stops at this point and waits until the job is completed. Applications like [Tasklist](/components/tasklist/introduction-to-tasklist.md) can be used by humans to complete these tasks. In this last step, you will open Tasklist to run the user task you created.
+
+:::tip
+While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial. Within this example, we've included a form to demonstrate the completion of a user task.
+
+Using the Zeebe or Tasklist API, many other ways to complete a user task are possible, such as redirecting to another application to complete the task, or even listening to IoT devices to capture human interaction with the real world via job workers.
```

Comment:
Would again provide links to this documentation -- how can I learn about these other ways to complete a user task?

Reply:
Is it ok to link to these at the end of the page? C.f. https://github.com/camunda/camunda-platform-docs/pull/2465#discussion_r1302128451

Reply:
I am okay with this 👍